workloads/baseline.yml

```diff
@@ -58,7 +58,7 @@
 
     - name: Check if scale-ci-tooling namespace exists
       shell: |
-        oc get projects | grep "scale-ci-tooling"
+         oc get project scale-ci-tooling
```

Comment:
nitpick but extra space in front of the `oc` command here.  Each indent should be 2 spaces